release_notes/Service_Fabric_ReleaseNotes_10.md

```diff
@@ -0,0 +1,76 @@
+# Microsoft Azure Service Fabric 10.0 Release Notes
+
+* [Service Fabric Packages and Versions](#service-fabric-packages-and-versions)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
```

Comment:
Add a Key Announcements section here on the top. Refer to https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_90.md as an example.